app/presenters/scoped_search_results_presenter.rb

```diff
@@ -0,0 +1,58 @@
+class ScopedSearchResultsPresenter < SearchResultsPresenter
+
+  def to_hash
+    super.merge({
+      is_scoped?: true,
+      scope_title: scope_title,
+      unscoped_results_any?: unscoped_results.any?,
+      unscoped_result_count: result_count_string(unscoped_result_count),
+      results: presentable_result_list,
+    })
+  end
+
+private
+
+  attr_reader :unscoped_results
+
+  def filter_fields
+    []
+  end
+
+  def scope_title
+    search_response["scope"]["title"]
+  end
+
+  def presentable_result_list
```

Comment:
I wonder if this would be better as an override of the `results` method?
```ruby
def results
  presentable_result_list = super # possibly a better local var name than this, but I'm currently unable to brain
  
  if unscoped_results.any?
    insertion_point = [results.count, 3].min
    unscoped_results_sublist = { results: unscoped_results, is_multiple_results: true }
     presentable_result_list.insert(insertion_point, unscoped_results_sublist)
  end
  presentable_result_list
end
```

Then you wouldn't need to change `results` in `to_hash` above, and intent is clearer.